crates/matrix-sdk/src/sliding_sync.rs

```diff
@@ -585,7 +586,14 @@ pub struct SlidingSync {
     /// keeping track of retries and failure cunts
     failure_count: Arc<AtomicU8>,
 
-    extensions: Mutable<Option<ExtensionsConfig>>,
+    /// the initial set of extensions used for this sliding sync instance - this
+    /// is what we reset the extensions to when the session expires.
+    initial_extensions: Option<ExtensionsConfig>,
```

Comment:
I think this is now dead code since we never `take()` the value from the `extensions` field.